spec/factories/packages.rb

```diff
@@ -27,11 +27,31 @@
 
     trait :package_with_locations do
       after(:create) do |package|
-        package_location = create :packages_location, package_id: package.id, quantity: package.received_quantity
-        package.location_id = package_location.location_id
-        package.packages_locations ||= []
-        package.packages_locations << package_location
-        package.save
+        build(:packages_location, package_id: package.id, quantity: package.received_quantity).sneaky do |package_location|
```

Comment:
sneaking sneaky into specs now I see :+1:  :100:  :smile:

Reply:
Didn't want a factory to secretly trigger builds of other tables 😄